clientgui/sg_ProjectPanel.cpp

```diff
@@ -298,7 +298,7 @@ void CSimpleProjectPanel::ReskinInterface() {
 
     CMainDocument* pDoc = wxGetApp().GetDocument();
     ProjectSelectionData* selData;
-    PROJECT* project;
+    PROJECT* project __attribute__((unused));
```

Comment:
I believe this line and line 311 `project = pDoc->state.lookup_project(ctrl_url);` might be safely removed since they are doing nothing